govcd/metadata.go

```diff
@@ -91,58 +91,16 @@ func (vapp *VApp) GetMetadata() (*types.Metadata, error) {
 	return getMetadata(vapp.client, vapp.VApp.HREF)
 }
 
-func getMetadata(client *Client, requestUri string) (*types.Metadata, error) {
```

Comment:
Just moved these private functions to the bottom for the sake of ordering a bit.